package.json

```diff
@@ -34,6 +34,7 @@
     "@pabra/sortby": "^1.0.1",
     "axios": "^0.21.4",
     "buffer": "^6.0.3",
+    "idb-keyval": "^6.2.0",
```

Comment:
New package that wraps and simplifies usage of IndexedDB for local storage handling.